.github/workflows/deploy-infra.yaml

```diff
@@ -206,26 +212,37 @@ jobs:
         run: |-
           sudo wget https://github.com/mikefarah/yq/releases/download/3.4.1/yq_linux_amd64 -O /usr/local/bin/yq && sudo chmod +x /usr/local/bin/yq
 
-      - id: fill-account-specific-metadata
-        name: Fill in account specific metadata in ConfigMap
+      - id: setup-domain
+        name: Compile environment-specific domain name
         run: |-
           if [ "${{ matrix.environment }}" = "production" ]; then
-            DOMAIN_NAME="scp.biomage.net"
+            DOMAIN_NAME=${{ secrets.DOMAIN_NAME_PRODUCTION }}
           fi
 
           if [ "${{ matrix.environment }}" = "staging" ]; then
-            DOMAIN_NAME="scp-staging.biomage.net"
+            DOMAIN_NAME=${{ secrets.DOMAIN_NAME_STAGING }}
           fi
 
-          AWS_REGION="${{ env.region }}"
-          AWS_ACCOUNT_ID="${{ steps.setup-aws.outputs.aws-account-id }}"
+          if [ -z "$DOMAIN_NAME" ]; then
+            echo "No domain name provided. Set DOMAIN_NAME_PRODUCTION or DOMAIN_NAME_STAGING in repository secrets."
+            exit 1
```

Comment:
could you use this error message in line 59 too? It is better than what you have on line 59 at the moment

Reply:
On 2nd thoughts, we don't actually need this check if we have the check on line 59. I'll check that the guarantee applies. If it is so, I'll remove this line.